### ISSUE TITLE
SSO as primary login module

### ISSUE THREAD

**Issue by prochan2** [JetBrains/teamcity-azure-active-directory#41]
Is it possible to use SSO by default and open the built-in login page on demand only?

**Comment by laurencee** [comment]
Yeah this would be pretty useful given the general move towards cloud in the industry.

**Comment by ddoadmin** [comment]
i am in process of implementing SSO and this is blocker for me, it would be nice to have this option in the configuration it self, for e.g. a switch to on and off SSO as default option for authentication.

**Comment by OleksiiH** [comment]
Any updates on this?
Very required functionality

**Comment by GalballyJ** [comment]
this would also be helpful for us

**Comment by alexrudd** [comment]
In the authentication settings there is now this option:

![image](https://github.com/JetBrains/teamcity-azure-active-directory/assets/3588652/71e6da74-e33f-49c4-a823-0fe3fce6f26a)

Which then presents a login screen like so:

![image](https://github.com/JetBrains/teamcity-azure-active-directory/assets/3588652/5e3fa506-0fe2-4ff6-a83a-52b348afacc3)
